app/models/providers/infra/vm_or_template.rb

```diff
@@ -51,5 +74,15 @@ def self.post_refresh_ems(ems_id, update_start_time)
         end
       end
     end
+
+    private
+
+    def power_state=(new_power_state)
```

Comment:
why do we need overriding this?

Reply:
No clue, just copied from manageiq
I assume some provider wants to control saving power_state directly?

Reply:
ic!